frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -346,7 +346,7 @@ const PairElement = DragSource(DRAGGABLE_TYPE.ENV_ROW, pairSource, collectSource
               </div>
               :
               <div className="col-xs-5 pairs-list__value-field">
-                <input type="text" className="pf-c-form-control" placeholder={valueString.toLowerCase()} value={pair[NameValueEditorPair.Value] || ''} onChange={this._onChangeValue} disabled={readOnly} />
+                <input type="text" className="pf-c-form-control" placeholder={valueString.toLowerCase()} value={pair[NameValueEditorPair.Value] || ''} onChange={this._onChangeValue} disabled={readOnly} data-test-id="nameValueEditorKey" />
```

Comment:
Why is the `data-test-id` `nameValueEditior` for both name and value input fields?

Reply:
It was a typo. Fixed it.